src/quetoo.h

```diff
@@ -250,6 +250,9 @@ typedef enum {
 #define QUETOO_TICK_SECONDS	(1.0 / QUETOO_TICK_RATE)
 #define QUETOO_TICK_MILLIS	(1000 / QUETOO_TICK_RATE)
 
+#define QUETOO_TO_MILLIS(t)	(t * 1000)
```

Comment:
`SECONDS_TO_MILLIS` / `MILLIS_TO_SECONDS`, please. `QUETOO_` is not a unit of time :)

Reply:
Quetoo Seconds are a unit of time though!

Reply:
`SECONDS_TO_MILLIS`